.github/ISSUE_TEMPLATE/request-for-position.yml

```
@@ -0,0 +1,84 @@
+name: Request for WebKit's position on an emerging web specification
+description: If you're working on a spec and would like to know the WebKit Open Source Projet's position on it, this is the form for you!
+body:
+  - type: markdown
+    attributes:
+      value: "## WebKittens who can provide input"
+  - type: input
+    id: webkittens
+    attributes:
+      label: "WebKittens"
+      description: "Who from the WebKit community do you think is best positioned to review this spec? It's okay if you don't know."
+      placeholder: "@annevk @hober @marcoscaceres"
+  - type: markdown
+    attributes:
+      value: "## Information about the specification or proposal"
+  - type: input
+    id: spec-title
+    attributes:
+      label: "Title of the spec"
+      placeholder: "CSS Frobnication Module, Level 5"
+    validations:
+      required: true
+  - type: input
+    id: spec-link
+    attributes:
+      label: "URL to the spec"
+      placeholder: "https://www.w3.org/TR/SHORTNAME"
+    validations:
+      required: true
+  - type: input
+    id: spec-repo
+    attributes:
+      label: "URL to the spec's repository"
+      description: "On GitHub or whatever repository hosting provider the spec uses."
+      placeholder: "https://github.com/ORG/REPO"
```

As we used W3C above... 

```suggestion
      placeholder: "https://github.com/w3c/REPO"
```